crates/release_plz_core/src/command/release.rs

```diff
@@ -170,18 +170,22 @@ impl ReleaseRequest {
         config.features.clone()
     }
 
-    fn registry_token(&self) -> Option<secrecy::Secret<String>> {
-        self.registry
-            .clone()
-            .and_then(|r| {
-                // Credentials for a specific registry can be set using environment variables.
-                // https://doc.rust-lang.org/cargo/reference/config.html#credentials
-                let env_token = env::var(format!("CARGO_REGISTRIES_{}_TOKEN", r.to_uppercase()))
-                    .ok()
-                    .map(SecretString::new);
-                self.token.clone().or(env_token)
-            })
-            .or(self.token.clone())
+    fn find_registry_token(
+        &self,
+        registry: &Option<String>,
+    ) -> anyhow::Result<Option<secrecy::Secret<String>>> {
+        if self.registry == *registry {
```

Comment:
this is also true if both are None. Is it expected?
Also, can you document a bit this function?
Thanks

Reply:
yes, it's expected. None means `crates.io` and I guess `--token` could be used also in this case and fallback to `crates:token`.